### PR TITLE
fix(translations): fix wrong Estonian (et) translations and typos

### PR DESCRIPTION
**Pull request by Gekd**
## Pre-flight checklist

- [x] I have read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/main/CONTRIBUTING.md#pull-requests).
- [ ] **If this is a code change**: I have written unit tests and/or added dogfooding pages to fully verify the new behavior.
- [ ] **If this is a new API or substantial change**: the PR has an accompanying issue (closes #0000) and the maintainers have approved on my working plan.

## Motivation

Fix Estonian default translations.

## Related issues/PRs

https://github.com/facebook/docusaurus/issues/3526

### CODE REVIEW

File: packages/docusaurus-theme-translations/locales/et/theme-search-algolia.json [hunk 1]
```diff
@@ -32,4 +32,4 @@
   "theme.SearchPage.inputLabel": "Otsi",
   "theme.SearchPage.inputPlaceholder": "Kirjuta oma päring siia",
   "theme.SearchPage.noResultsText": "Tulemusi ei leitud"
-}
```
Comment (slorber) [inline packages/docusaurus-theme-translations/locales/et/theme-search-algolia.json:35]: please remove the extra line breaks here, our linter won't le me merge with it

**Review by slorber — CHANGES_REQUESTED** [review]
Thanks,

LGTM but I don't speak Estonian 😅 

Little linting issues

This language has only been added recently. Can you help review it @chirbard please?

File: packages/docusaurus-theme-translations/locales/et/theme-live-codeblock.json [hunk 2]
```diff
@@ -1,4 +1,4 @@
 {
-  "theme.Playground.liveEditor": "Live Redaktor",
+  "theme.Playground.liveEditor": "Elav Redaktor",
```
Comment (chirbard) [inline packages/docusaurus-theme-translations/locales/et/theme-live-codeblock.json:2]: I'd say "Live Redaktor" is more correct.

File: packages/docusaurus-theme-translations/locales/et/theme-search-algolia.json [hunk 3]
```diff
@@ -1,15 +1,15 @@
 {
   "theme.SearchBar.label": "Otsi",
-  "theme.SearchBar.seeAll": "Näe kõiki {count} tulemust",
+  "theme.SearchBar.seeAll": "Vaata kõiki {count} tulemust",
   "theme.SearchModal.errorScreen.helpText": "Kontrolli interneti ühendust.",
   "theme.SearchModal.errorScreen.titleText": "Tulemuste pärimine ebaõnnestus",
-  "theme.SearchModal.footer.closeKeyAriaLabel": "Escape nuppu",
+  "theme.SearchModal.footer.closeKeyAriaLabel": "Paoklahv nuppu",
```
Comment (chirbard) [inline packages/docusaurus-theme-translations/locales/et/theme-search-algolia.json:6]: "Paoklahvi"

File: packages/docusaurus-theme-translations/locales/et/theme-search-algolia.json [hunk 4]
```diff
@@ -1,15 +1,15 @@
...
   "theme.SearchModal.footer.closeText": "sulgemiseks",
   "theme.SearchModal.footer.navigateDownKeyAriaLabel": "Alla noolt",
   "theme.SearchModal.footer.navigateText": "navigeerimiseks",
   "theme.SearchModal.footer.navigateUpKeyAriaLabel": "Üles noolt",
   "theme.SearchModal.footer.searchByText": "Otsi",
-  "theme.SearchModal.footer.selectKeyAriaLabel": "Enter nuppu",
+  "theme.SearchModal.footer.selectKeyAriaLabel": "Sisestusklahv nuppu",
```
Comment (chirbard) [inline packages/docusaurus-theme-translations/locales/et/theme-search-algolia.json:12]: "Sisestusklahvi"

**Review by chirbard — COMMENTED** [review]
Added some comments that should be fixed. Everything else looks good to me.

**Review by Gekd — COMMENTED** [review]
Fixed your comments.

**Review by chirbard — COMMENTED** [review]
LGTM

**Comment by slorber** [comment]
@Gekd thanks

We need you to sign the CLA. I can't merge the PR otherwise.

**Comment by Gekd** [comment]
Signed CLA.

**Pull request merged**